### ISSUE TITLE
Product Search (editor): placeholder doesn't have enough contrast

### ISSUE THREAD

**Issue by Aljullu** [woocommerce/woocommerce-gutenberg-products-block#1683]
![imatge](https://user-images.githubusercontent.com/3616980/73442110-7bb51400-4354-11ea-8e08-034ac6448ffd.png)

Placeholder is `#828B96` while background is white, that combination doesn't have enough contrast:

https://webaim.org/resources/contrastchecker/?fcolor=828B96&bcolor=FFFFFF

We should replace it with another color. Ideally, one in our `_colors.scss` file:

https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/assets/css/abstracts/_colors.scss

Or one of the colors we consume upstream:

https://github.com/WordPress/gutenberg/blob/trunk/packages/base-styles/_colors.scss
https://github.com/Automattic/color-studio/blob/master/dist/color-variables.scss

**Comment by Aljullu** [comment]
Heads-up that we are working on completely refactoring the Product Search block (https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/6145) so hopefully this issue won't be reproducible anymore once that's finished.
